README.md

```diff
@@ -14,6 +14,70 @@ by leveraging the modern web.
 ## API Spec
 The preferred JSON object to be returned by the API should be structured as follows:
 
+
+### Author's Haven API on heroku
+[Author's Haven API](https://ivy-ah-backend.herokuapp.com/)
+
+### Author's Haven API documentation
+[Author's Haven documentation](https://ivy-ah-backend.herokuapp.com/api-docs)
+
+
+### How To Install and Run This Application on your Computer
+1. Download and install [GIT](https://git-scm.com/downloads), [NodeJs](https://nodejs.org/en/) and [Postgresql](https://www.postgresql.org/download/) on your computer
+2. Create a database with the name authorshaven (This will be part of DATABASE_URL in the .env file). 
+3. Clone this repository by running the following on your command line interface
+`
+git clone https://github.com/andela/ivy-ah-backend.git
+`
+4. Navigate to the directory where the app was cloned to with `cd ivy-ah-backend`
+5. Run `npm install` to install all dependencies
+6. Rename the `.env-example` file to `.env` and set all the environment variables
+7. For the SENDGRID_API_KEY variable, create an account on [Sendgrid](sendgrid.com) to get the API KEY
```

Comment:
Link redirects to 404 page on GitHub. You'd need to format it correctly

Reply:
OK, I will correct that now. Thanks.